src/mixin/java/io/github/brassmc/brassloader/mixin/BootstrapMixin.java

```diff
@@ -0,0 +1,48 @@
+package io.github.brassmc.brassloader.mixin;
+
+import io.github.brassmc.brassloader.boot.discovery.InvalidEntrypointException;
+import io.github.brassmc.brassloader.boot.discovery.ModDiscovery;
+import io.github.brassmc.brassloader.boot.mods.ModContainer;
+import net.minecraft.server.Bootstrap;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.util.Arrays;
+
+@Mixin(Bootstrap.class)
+public class BootstrapMixin {
+    @Inject(
+            at = @At(
+                    value = "INVOKE",
+                    target = "Lnet/minecraft/core/Registry;freezeBuiltins()V"
+            ),
+            method = "bootStrap"
+    )
+    private static void init(CallbackInfo callback) {
```

Comment:
Also, no idea how I missed it before, but brass prefix. 
```suggestion
    private static void brass$loadMods(CallbackInfo callback) {
```